inst/ms/aggreCAT.qmd

```diff
@@ -400,39 +423,43 @@ for a single claim to illustrate this structure of the core
 `data_ratings` dataset.
 
 ```{r}
-#| label: data_ratings
+#| label: data_ratings-sample
 #| message: false
 #| prompt: true
+#| results: hold
+
 library(tidyverse,quietly = TRUE)
 library(aggreCAT)
 data(data_ratings)
 data_ratings %>% 
-  print(n = 18)
+  dplyr::filter(paper_id == dplyr::first(paper_id), 
+                user_name == dplyr::first(user_name)) %>% 
+  print(., n = nrow(.))
```

Comment:
Is there a way to ensure the `element` column isn't being truncated? It's one of the more important columns to see in full.

Also, `group` is shown in the output but not define in text above.

Reply:
Stopped truncation of `element` by changing output code width.
Explained what 'group' column is and that the user can include additional columns as needed.